### PR TITLE
Run3-gex83A Make a few SD's in SimG4CMS/Forward with the new way of initiation

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Make a few SD's in SimG4CMS/Forward with the new way of initiation

#### PR validation:

Use the runTheMatrix test workflows

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
@bsunanda , it is not a good idea renaming these classes. Previous names were fine, what is bad in CastorSD?

**Comment by civanch** [comment]
@bsunanda , we need to complete migration of SD classes to the new design. Chris made this only for main par, there several SD classes to be migrated. We need to find all (used or not used and do this.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: SimG4CMS/Forward/src/TotemSD.cc [hunk 1]
```diff
@@ -42,11 +42,11 @@
 #include "G4PhysicalConstants.hh"
 #include "G4SystemOfUnits.hh"
 
+#define EDM_ML_DEBUG
```
Comment (civanch) [inline SimG4CMS/Forward/src/TotemSD.cc:45]: @bsunanda , please, comment out DEBUG.

File: SimG4CMS/Forward/src/ZdcSD.cc [hunk 2]
```diff
@@ -25,8 +25,9 @@
 #include "Randomize.hh"
 #include "G4Poisson.hh"
 
+#define EDM_ML_DEBUG
```
Comment (civanch) [inline SimG4CMS/Forward/src/ZdcSD.cc:28]: Here as well.

File: SimG4CMS/Forward/src/TotemSD.cc (hunk 1 above)
Comment (bsunanda) [reply SimG4CMS/Forward/src/TotemSD.cc]: Done

File: SimG4CMS/Forward/src/ZdcSD.cc (hunk 2 above)
Comment (bsunanda) [reply SimG4CMS/Forward/src/ZdcSD.cc]: Done

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**